src/Bicep.LangServer.IntegrationTests/TelemetryTests.cs

```diff
@@ -339,18 +327,171 @@ public async Task BicepFileOpen_ShouldFireTelemetryEvent()
                     // location: resourceGroup().location
                     //   => Use a parameter here instead of 'resourceGroup().location'. 'resourceGroup().location' and 'deployment().location' should only be used as a default value for parameters.
                     "warnings",
-                    "2"
+                    "1"
                 },
                 { "modulesInReferencedFiles", "0" },
                 { "parentResourcesInReferencedFiles", "2" },
                 { "parametersInReferencedFiles", "2" },
                 { "variablesInReferencedFiles", "1" },
-                { "lineCountOfReferencedFiles", "12" }
+                { "lineCountOfReferencedFiles", "12" },
+                { "disableNextLineDirectivesCount", "4" },
+                { "diagnosticCodesInDisableNextLineDirectives", "{\"no-hardcoded-location\":2,\"BCP036\":1,\"BCP037\":1}" },
```

Comment:
Really no need to be so explicit and long.  

suggest:
disableNextLineCount (this can be retrieved from disableNextLineCodes, although slower that way)
disableNextLineCodes
disableNextLineCountInReferencedFiles
disableNextLineCodesInReferencedFiles

Reply:
Updated as part of this commit:
https://github.com/Azure/bicep/pull/6151/commits/e851cc997271d236fcc23ce2f23963c1d40e71c8

Thanks!